src/pathfinding_service/service.py

```diff
@@ -285,56 +285,88 @@ def create_token_network_for_address(
 
     def on_pfs_update(self, message: UpdatePFS):
+        token_network_address = to_checksum_address(
+            message.canonical_identifier.token_network_address,
+        )
         log.info(
-            'Received Balance Update',
-            token_network_address=message.token_network_address,
-            channel_identifier=message.channel_identifier,
+            'Received Capacity Update',
+            token_network_address=token_network_address,
+            channel_identifier=message.canonical_identifier.channel_identifier,
         )
 
         assert is_checksum_address(message.updating_participant)
         assert is_checksum_address(message.other_participant)
 
         # check if chain_id matches
-        if message.chain_id != self.chain_id:
-            log.info(
-                'Received balance update with unknown chain id',
-                chain_id=message.chain_id,
-            )
-            return
+        if message.canonical_identifier.chain_identifier != self.chain_id:
+            raise InvalidCapacityUpdate('Received Capacity Update with unknown chain identifier')
 
         # check if token network exists
-        token_network = self._get_token_network(message.token_network_address)
+        token_network = self._get_token_network(token_network_address)
         if token_network is None:
-            log.info(
-                'Received balance update with unknown token network',
-                token_network_address=message.token_network_address,
-            )
-            return
+            raise InvalidCapacityUpdate('Received Capacity Update with unknown token network')
 
         # check if channel exists
-        if message.channel_identifier not in token_network.channel_id_to_addresses:
-            log.info(
-                'Received balance update with unknown channel_id in token network',
-                channel_id=message.channel_identifier,
-                token_network_address=message.token_network_address,
+        channel_identifier = message.canonical_identifier.channel_identifier
+        if channel_identifier not in token_network.channel_id_to_addresses:
+            raise InvalidCapacityUpdate(
+                'Received Capacity Update with unknown channel identifier in token network',
             )
-            return
 
         # TODO: check signature of message
 
         # check values max int 256 TODO: get constants
-        if not 0 < message.updating_capacity < 2 ^ 256:
-            return
-        if not 0 < message.other_capacity < 2 ^ 256:
-            return
+        if not 0 < message.updating_capacity < UINT256_MAX:
+            raise InvalidCapacityUpdate(
+                'Received Capacity Update with impossible updating_capacity',
+            )
+        if not 0 < message.other_capacity < UINT256_MAX:
+            raise InvalidCapacityUpdate(
+                'Received Capacity Update with impossible other_capacity',
```

Comment:
nitpick: Here and other parts of the file, why capitalize the words capacity and update?

Reply:
We wanted to use "Capacity Update" as a term ... The message should be called Capacity Update. Does that make sense to you?

Reply:
Sure I get it. It just seems odd a bit. But it's fine. Just felt the need to ask if it's a typo/problem.

Reply:
Yes good that you pointed that out!